### PR TITLE
46770 QA and Live deployment stages

### PR DESCRIPTION
**Pull request by tejasi15072**
This PR contains QA and live environment deployment changes in azure-pipeline.yml file .

### CODE REVIEW

**Review by barrie-cooper — COMMENTED** [review]
Lines 378-464 are exactly the same as lines 266-352, apart from 'Dev' vs 'QA'.

Could we have the Functional and Accessibility tests moved into a template (or 2) please?

File: Deployment/templates/continuous-testing.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      workingDirectory: '$(Build.SourcesDirectory)/Tests/AutoTests/FunctionalTest'
+
+    - task: PublishTestResults@2
+      condition: succeededOrFailed()
+      inputs:
+        testRunner: 'JUnit'
+        testResultsFiles: 'results.xml'
+        mergeTestResults: true
+        searchFolder:  '$(System.DefaultWorkingDirectory)\Tests'
+        testRunTitle: Dev Functional Test Results
```
Comment (barrie-cooper) [inline Deployment/templates/continuous-testing.yml:32]: Could be QA 😮 
```
        testRunTitle: Dev Functional Test Results
```

File: Deployment/templates/continuous-testing.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      workingDirectory: '$(Build.SourcesDirectory)/Tests/AutoTests/AccessebilityTest'
+          
+    - task: PublishTestResults@2
+      condition: succeededOrFailed()
+      inputs:
+        testRunner: 'JUnit'
+        testResultsFiles: 'results.xml'
+        mergeTestResults: true
+        searchFolder:  '$(System.DefaultWorkingDirectory)\Tests'
+        testRunTitle: Dev Accessibility Test Results
```
Comment (barrie-cooper) [inline Deployment/templates/continuous-testing.yml:47]: Could be QA 😮 
```
        testRunTitle: Dev Accessibility Test Results
```

File: Deployment/templates/continuous-testing.yml (hunk 1 above)
Comment (tejasi15072) [reply Deployment/templates/continuous-testing.yml]: Environment name replace with environment variable.

File: Deployment/templates/continuous-testing.yml (hunk 2 above)
Comment (tejasi15072) [reply Deployment/templates/continuous-testing.yml]: environment name replace with environment variable.

**Comment by tejasi15072** [comment]
> Lines 378-464 are exactly the same as lines 266-352, apart from 'Dev' vs 'QA'.
> 
> Could we have the Functional and Accessibility tests moved into a template (or 2) please?

Suggested changes are done in the pipeline.

**Pull request merged**